libs/stache/src/lib/modules/nav/nav.component.spec.ts

```diff
@@ -84,41 +84,6 @@ describe('StacheNavComponent', () => {
     expect(component.navType).toBe('sidebar');
   });
 
-  it('should return true if the component has routes', () => {
```

Comment:
👇 These tests were deleted because they were testing private methods (now removed) simply for coverage's sake.